### PR TITLE
Test/cypress refactor

### PR DESCRIPTION
**Pull request by RosaTheDev**
x Wrote Tests ____ Implemented ____ Reviewed

## Necessary checkmarks:

    [] All Tests are Passing in all environments
    [x] Tests pass locally and in cypress but not on Circle CI 
    [] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [x] Refactor

## Description of Change:

    description: Refactored tests that were failing due to added code and functionality, and added tests to check for the bigfoot information, tested to make sure that there is a description and danger information available
    
## Requested feedback: 
    
    I'd like feedback on... Why does Circle CI hate this? 

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [] No Tests have been changed
    [x] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: cypress/integration/allsightings-test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+describe('All sightings page cypress test', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+  
+  it('Should confirm that true is equal to true', () => {
```
Comment (GraceGardner) [inline cypress/integration/allsightings-test.js:7]: Now that there are more tests, this sample test can be removed.

File: cypress/integration/allsightings-test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    cy.get('.cryptid-button')
+    cy.get('.zipcode-input')
+  })
+
+  it('Should display all the sightings of the cryptids', () => {
+    cy.get('button').click()
+    cy.get('.all-sightings-container')
+  })
+
+  it('Should click on logo and move to home page', () => {
```
Comment (GraceGardner) [inline cypress/integration/allsightings-test.js:30]: Could make more detailed by checking something exists on the home page to confirm navigation

File: cypress/integration/allsightings-test.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    cy.get('button').click()
+    cy.get('.logo').click()
+  })
+
+  it('Should click on ok, Cryptid and move to home page', () => {
+    cy.get('button').click()
```
Comment (GraceGardner) [inline cypress/integration/allsightings-test.js:36]: This test is the same as the one above I believe? Being both the logo and text are in the same container. Never hurts to be detailed though.

File: cypress/integration/home-test.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+describe('Home page cypress test', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+  
+  it('Should confirm that true is equal to true', () => {
```
Comment (GraceGardner) [inline cypress/integration/home-test.js:7]: This sample test could also be removed.

File: cypress/integration/home-test.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    expect(true).to.equal(true)
+    expect(true).to.equal(true)
+  });
+
+  it('Should render elements to the page', () => {
+    cy.get('.logo')
+    cy.get('.home-message')
+    cy.get('.home-card-container')
```
Comment (GraceGardner) [inline cypress/integration/home-test.js:15]: Could also check that there are elements inside the home container

File: cypress/integration/singleCryptid-test.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+describe('All sightings page cypress test', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+  
+  it('Should confirm that true is equal to true', () => {
```
Comment (GraceGardner) [inline cypress/integration/singleCryptid-test.js:7]: This sample test could also be removed.

File: cypress/integration/singleCryptid-test.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    expect(true).to.equal(true)
+    expect(true).to.equal(true)
+  });
+
+  it('Should start on the home page and click on a cryptid', () => {
+    cy.get('\#\\32').click()
+  })
+
+  it('Should view the cryptids information page', ()=> {
```
Comment (GraceGardner) [inline cypress/integration/singleCryptid-test.js:16]: Line 13 and 17 are really cool. Also like how detailed this is to check that it's navigating to a new page and making sure that everything is rendering.  Nice work

**Review by GraceGardner — APPROVED** [review]
Nice work. There are a few sample tests still in the code that could be removed and I might  just check something on the page after navigating to a new page, just to confirm that the links are working properly. All in all it looks great!

**Review by krdurrett — APPROVED** [review]
Agree with Grace, sample tests can be removed

**Pull request merged**